x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -417,6 +418,15 @@ export const RulesList = ({
 
   const handleClearRuleParamFilter = () => updateFilters({ filter: 'ruleParams', value: {} });
 
+  usePageReady({
+    isReady: !rulesState?.initialLoad,
+    isRefreshing: rulesState?.isLoading,
+    meta: {
+      description:
+        '[ttfmp_rules_list] The Observability Rules List overview page has loaded successfully.',
```

Comment:
Linked to the previous one, wouldn't this also be the stack rules page?

Reply:
You're right the description will be common, the stack rules page vs o11y rules page should be differentiable from the path and the other fields reported. I'll change the description to be more generic